Added test Framework Utility  Functions For the json Schema (#1078)
jenkinsci/configuration-as-code-plugin: docs/PLUGINS.md

    }
}
```

> ### JSON Schema Test

Reviewer: ```suggestion
### JSON Schema Test (Beta)
```

> The new Schema can be used to validate the provided YAML file describing the configurators of the corresponding

Reviewer: maybe something like:
```suggestion
We generate a JSON schema that users can use to validate their changes and provide IDE assistance, you can test that your plugin's example yaml file validates correctly by implementing the below test:
```

plugin.
`SchemaGenerationTest` provides a abstraction layer to test out the plugins YAML file against the generated schema.

> Hence Developers can now test if their YAML file validates against the schema.

Reviewer: ```suggestion
You can test if their YAML file validates against the schema.
```

> ### Step 1

Reviewer: ```suggestion
#### Step 1
```

> #### Step 1

Reviewer: ```suggestion
Step 1
```

Create a YAML file for the configurators corresponding to the developed plugin.
For eg: `validJenkinsConfigurator.yml`
```
jenkins:
    systemMessage: "Configured by Configuration as Code plugin"
    numExecutors: "Hello"

```

> ### Step 2

Reviewer: ```suggestion
#### Step 2
```

> #### Step 2

Reviewer: ```suggestion
Step 2
```

Add a test for the YAML file

> ```

Reviewer: ```suggestion
```java
```

> @Test

Reviewer: formatting is off:
![image](https://user-images.githubusercontent.com/21194782/66395416-4b292300-e9cf-11e9-91b2-021bf48a12de.png)
